File: 1 (6.23)/1152/1152.java
// 문자열의 단어 개수 세기
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int count = 0;
        String[] str = sc.nextLine().trim().split(" ");

        for (int i = 0; i < str.length; i++) {
            if (!str[i].equals("")) {
                count++;
            }
        }

        System.out.print(count);
    }
}

File: 1 (6.23)/1152/1152.ts
// 문자열의 단어 개수 세기

let count = 0;
const inputs = require('fs').readFileSync('/dev/stdin').toString().trim().split(' ');

for (let i = 0; i < inputs.length; i++) {
    if (inputs[i] !== '') {
        count++;
    }
}

console.log(count);